### PR TITLE
More declarative, and new variant

### PR DESCRIPTION
**Pull request by mimse**
## Description

Made the description list more declartive to use. Also added variant. So we now have the noarmal flow, and a stack version.

## References
https://github.com/Energinet-DataHub/greenforce-frontend/issues/1506

### CODE REVIEW

**Review by FirestarJes — COMMENTED** [review]
Looks fine to me, but a more experienced frontender should probably also take a look at it.

File: libs/dh/wholesale/feature-search/src/lib/batch-details/dh-wholesale-batch-details.component.ts [hunk 1]
```diff
@@ -75,29 +73,31 @@ export class DhWholesaleBatchDetailsComponent {
...
+  //         : '-',
+  //     },
+  //     {
+  //       term: translate('wholesale.batchDetails.executionTime'),
+  //       description: this.batch?.executionTimeStart
+  //         ? (dateTimePipe.transform(this.batch?.executionTimeStart) as string)
+  //         : '-',
+  //     },
+  //   ];
+  // }
```
Comment (dzhavat) [inline libs/dh/wholesale/feature-search/src/lib/batch-details/dh-wholesale-batch-details.component.ts:100]: question: any reason for keeping this code?

File: libs/dh/wholesale/feature-calculation-steps/src/lib/results/dh-wholesale-results.component.html [hunk 2]
```diff
@@ -30,10 +30,23 @@
...
+    <watt-description-list-item
+      [label]="t('meteringPointType')"
+      [value]="t('timeSeriesType.' + processStepResults.timeSeriesType)"
+    ></watt-description-list-item>
+    <watt-description-list-item
+      [label]="t('calculationPeriod')"
+      [value]="batch?.period | dhDateRange"
+    >
+    </watt-description-list-item>
+    <watt-description-list-item [label]="t('sum')" [value]="processStepResults.sum + ' kWh'">
```
Comment (frontend-specialisten) [inline libs/dh/wholesale/feature-calculation-steps/src/lib/results/dh-wholesale-results.component.html:43]: Question: In the old implementation, we can force a new row. Do you know how we can do this here? Should this be done by having two lists or a property on the `watt-description-list-item` component?

**Review by frontend-specialisten — COMMENTED** [review]
Praise: nice changes! I only have a single question and also the build needs to be fixed :)

File: libs/dh/wholesale/feature-calculation-steps/src/lib/results/dh-wholesale-results.component.html (hunk 2 above)
Comment (mimse) [reply libs/dh/wholesale/feature-calculation-steps/src/lib/results/dh-wholesale-results.component.html:43]: You can set forceNewRow=true on a list item

Comment (frontend-specialisten) [reply libs/dh/wholesale/feature-calculation-steps/src/lib/results/dh-wholesale-results.component.html:43]: Perfect!

**Review by dzhavat — APPROVED** [review]

**Pull request merged**